src/types/Status.ts

```diff
@@ -1 +1,5 @@
-export type Status = 'all' | 'active' | 'completed';
+export enum Status {
+  all = 'all',
+  active = 'active',
+  complited = 'completed',
```

Comment:
It's better to capitalize Enum properties
https://www.typescriptlang.org/docs/handbook/enums.html